src/navigator_data_ingest/base/actions.py

```diff
@@ -51,9 +55,11 @@ def handle_all_documents(
     The remote filename follows the template on
     https://www.notion.so/climatepolicyradar/Document-names-on-S3-6f3cd748c96141d3b714a95b42842aeb
     """
+    token = get_machine_user_token()
     tasks = {
         executor.submit(
             _handle_document,
+            token,
```

Comment:
I can't remember how long our access tokens are valid for, hopefully these tokens won't expire if we upload lots of documents!

Reply:
Ooo - good point to raise :thinking: